packages/styletron-preact/package.json

```diff
@@ -24,7 +24,7 @@
     "styletron-utils": "^2.5.4"
   },
   "peerDependencies": {
-    "preact": "6.x"
+    "preact": "8.x"
```

Comment:
Should we change this to `>=6`? I'm not sure there's anything that requires Preact 8